### PR TITLE
Allow tensorflow-sys to download prebuilt windows releases.

### PR DESCRIPTION
**Pull request by photex**
Howdy!

This appears to work as intended, but please let me know what you'd like to have changed. Since there are pre-built releases available for windows I figured it'd be worthwhile to also support them in this crate.

Cheers!
Chip

### CODE REVIEW

File: tensorflow-sys/build.rs [hunk 1]
```diff
@@ -94,13 +97,43 @@ fn remove_suffix(value: &mut String, suffix: &str) {
...
+fn extract<P: AsRef<Path>, P2: AsRef<Path>>(archive_path: P, extract_to: P2) {
+    fs::create_dir_all(&extract_to)
+        .expect("Failed to create output path for zip archive.");
+    let file = File::open(archive_path)
+        .expect("Unable to open libtensorflow zip archive.");
+    let mut archive = ZipArchive::new(file).unwrap();
+    for i in 0..archive.len() {
+        let mut zipfile = archive.by_index(i).unwrap();
+        let output_path = extract_to.as_ref().join(zipfile.sanitized_name());
+        if (&*zipfile.name()).starts_with("lib") {
```
Comment (adamcrume) [inline tensorflow-sys/build.rs:118]: Can't this be simplified to just `zipfile.name().starts_with("lib")`?

**Comment by adamcrume** [comment]
Please ensure that test-all runs successfully.  (There's supposed to be a Travis CI check, but it's not running for some reason.)

File: tensorflow-sys/Cargo.toml [hunk 2]
```diff
@@ -20,6 +20,7 @@ libc = "0.2.43"
 [build-dependencies]
 curl = "0.4.24"
 flate2 = "1.0.12"
+zip = "0.5"
```
[the remark this answers is not in the document]
Comment (photex) [reply tensorflow-sys/Cargo.toml]: No problem

File: tensorflow-sys/build.rs (hunk 1 above)
Comment (photex) [reply tensorflow-sys/build.rs]: It sure can. Can't recall what sequence of silliness led me to that anyway. :D

**Comment by photex** [comment]
Since I'm attempting to build and work with this without mingw or bash etc I'll have to do the work of ./test-all by hand.
I'm happy to put together a powershell or batch file if you think that's more suitable.

I also just realized that when I tested my changes I didn't actually try to use the crate in my app (as in `use tensorflow`, so this current state is incorrect. 

Changes incoming.

**Comment by photex** [comment]
Alrighty @adamcrume I ran the work in ./test-all manually in Windows and everything worked.

I then ran it in WSL and things worked fine as well. I don't have a MacOS machine to test with otherwise I'd be happy to also check there.

**Comment by adamcrume** [comment]
Thanks!  I always appreciate maintenance for the Windows build.

**Pull request merged**